google/cloud/firestore_v1/base_query.py

```diff
@@ -756,9 +760,10 @@ def _normalize_cursor(self, cursor, orders) -> Optional[Tuple[Any, Any]]:
                 except KeyError:
                     msg = _MISSING_ORDER_BY.format(order_key, data)
                     raise ValueError(msg)
+
             document_fields = values
 
-        if len(document_fields) != len(orders):
+        if len(document_fields) > len(orders):
```

Comment:
Nit: Throw this first to skip the work above

Reply:
I wanted to do that but it would require a bit more restructuring (documentsnapshots get converted to dicts to share this code), this is then used for both cases there and if it was an array.